### ISSUE TITLE
Remembering doesn't say "all locations have been explored" any more

### ISSUE THREAD

**Issue by mciul** [i7/kerkerkruip#244]
I know there were some changes to the remember command recently. Has the "all locations have been explored" message been broken?

**Comment by mciul** [comment]
I think I finally caught the Mausoleum bug! It turns out that the Mausoleum could be connected to a hidden room, in which case its hidden status was ambiguous. I have prevented that possibility in the branch I'm working on, which will hopefully see the light of the day sometime this summer.

**Comment by VictorGijsbers** [comment]
Great. :-) What is the branch you're working on?

**Comment by mciul** [comment]
It's linked in the commit above - https://github.com/i7/kerkerkruip/compare/ihateyougit_outcomes_only

I did a complete overhaul of the testing system - I took a small element (randomized outcomes) and based they entire system on them. Now you define outcomes in a table and they can have dependencies which are automatically tracked. I think it'll be simpler in the long run. I'm getting close to being able to merge - I have only 12 test sets left to update.
